src/lib/eliom_shared.eliom

```diff
@@ -19,14 +19,17 @@
  *)
 
 [%%shared
+open Lwt.Syntax
+
 (* put this in Lwt_react? Find a better name? *)
 let to_signal ~init ?eq (th : 'a React.S.t Lwt.t) : 'a React.S.t =
   let s, set = React.S.create ?eq init in
   Lwt.async (fun () ->
-    let%lwt ss = th in
-    let effectful_signal = React.S.map (fun v -> set v) ss in
-    ignore (React.S.retain s (fun () -> ignore effectful_signal));
-    Lwt.return_unit);
+      let* ss = th in
+
```

Comment:
There seems to be a white line after many let*. And indentation is different. Did you ocamlformat everything?

Reply:
Sadly ocamlformat doesn't seem to remove those newlines. I've updated by hand.